interfaces/HO-Portal/src/app/program/table-filter-row/table-filter-row.component.scss

```
@@ -38,11 +41,30 @@
 }
 
 .tfr--active-filters {
+  padding: 8px;
+
   ion-chip {
     cursor: default; // Make appear as unclickable
   }
 
   ion-chip ion-button {
     cursor: pointer; // Re-enable cursor to appear interactive normally
+    position: relative;
+    inset-inline-end: -0.5em;
+  }
+
+  .tfr--active-filters-component {
+    --background: var(--fiveten-purple-100);
+    border: 1px solid var(--fiveten-purple-200);
+    --color: var(--fiveten-neutral-900);
+    font-size: 12px;
+
+    &.tfr--active-filters-chip {
+      height: 2.2em;
```

```suggestion
      min-height: 2.2em;
```

So that we could potentially support "multi-line chips"? Or at least not make that impossible.